### DISCUSSION TITLE
List "Uncategorized" links

### DISCUSSION THREAD

**Discussion by rixtox** [Kovah/LinkAce#549] [Support] [answered]
There are times where you don't have time to tag/categorized links you added, and you may later want to sort them into lists and tag them. It would be great to have a view to quickly do this. Just listing everything that doesn't belong to a list or has a tag should be enough.

**Comment by Kovah** [comment, marked as answer]
Hi, you can use the "without Tags" and "without Lists" options in the search. They currently require an additional search query, which you can set to any character that is included in any URL, like the dot: `.` 
<img width="998" alt="Preview of linkace search" src="https://user-images.githubusercontent.com/1816101/195780133-367dfd10-68f2-4ffb-b013-e0ebd8e2cf9b.png">
